### PR TITLE
Add weight column

### PR DESCRIPTION
**Pull request by rachelkang**
Implementing #24

### CODE REVIEW

File: onnxjs/tests/test_cases.json [hunk 1]
```diff
@@ -1,194 +1,242 @@
...
         "text": "This code is really bad",
-        "isnegative": "1"
+        "isnegative": "1",
+        "importance": "1"
     },
     {
         "text": "There are test failures",
-        "isnegative": "0"
+        "isnegative": "0",
+        "importance": "1"
```
Comment (jonathanpeppers) [inline onnxjs/tests/test_cases.json:20]: I don't think these lines in `test_cases.json` will do anything.

I think we need to figure out what to put here:

https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/e46e2cc81f2d2c13e41c7b06c559d5c31044ede1/onnxjs/tests/onnx.ts#L15-L18

I tried something like `importance: new ort.Tensor('float32', [1,1]),`, but it's not quite right:
```
Error: Invalid rank for input: importance Got: 1 Expected: 2 Please fix either the inputs or the model.
```

File: ml.net/InclusiveCodeReviews.Model/ModelInput.cs [hunk 2]
```diff
@@ -17,6 +17,9 @@ public class ModelInput
         [ColumnName("isnegative"), LoadColumn(1)]
         public string Isnegative { get; set; }
 
+        [ColumnName("importance"), LoadColumn(2)]
+        public float Importance { get; set; }
```
Comment (jonathanpeppers) [inline ml.net/InclusiveCodeReviews.Model/ModelInput.cs:21]: Should this be:
```suggestion
        [ColumnName("importance"), LoadColumn(2)]
        public float Importance { get; set; } = 0.5f;
```
Then we might test the MLTrainer app can add a row? And it just always puts 0.5 by default?

**Comment by rachelkang** [comment]
In general you should avoid this pattern because it often leads to memory leaks.

**Comment by rachelkang** [comment]
This is awful. Whoever taught you this should be fired.

**Pull request merged**